pkg/controller/replicaset/replica_set.go

```diff
@@ -37,8 +37,7 @@ import (
 	"time"
 
 	apps "k8s.io/api/apps/v1"
-	"k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/api/core/v1"
```

Comment:
nitpick: is added import alias needed here?

Reply:
It's autoformated by vscode-go plugin.I think it's better to use it as a explicit alias